syntax = "proto3";
// 请求消息
message RequestMsg
{
    int32 cmd_type = 1;            // 命令类型
    bytes client_id = 2;           // 客户端ID，
    bytes server_id = 3;           // 服务器ID
    bytes sign = 4;                // 签名
    bytes data = 5;                // 数据
}
// 响应消息
message RespondMsg
{
    bool status = 1;               // 状态
    int32 seckey_id = 2;           // 密钥ID
    bytes client_id = 3;           // 客户端ID
    bytes server_id = 4;           // 服务器ID
    bytes data = 5;                // 数据
}
